simulariumio/constants.py

```diff
@@ -127,3 +127,6 @@ def JMOL_COLORS() -> pd.DataFrame:
     """
     this_dir, _ = os.path.split(__file__)
     return pd.read_csv(os.path.join(this_dir, JMOL_COLORS_CSV_PATH))
+
+
+MAX_AGENT_ID = 0x80000000  # Agent IDs should fit in a 32 bit signed int
```

Comment:
just curious, why signed int and not unsigned? Also, strategically, is this a check on bad(corrupt or erroneous) data, or a check on scalability to numbers of agents we can't handle? Or something else?

Reply:
unsigned hypothetically would work. And we're checking scalability to make sure all the numbers can be handled by front end